sormas-e2e-tests/src/main/java/org/sormas/e2etests/utils/TestUtils.java

```diff
@@ -0,0 +1,40 @@
+package org.sormas.e2etests.utils;
+
+import io.cucumber.datatable.dependency.com.fasterxml.jackson.databind.DeserializationFeature;
+import io.cucumber.datatable.dependency.com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import javax.inject.Inject;
+import lombok.Getter;
+import lombok.Setter;
+import org.sormas.e2etests.state.BodyResources;
+
+public class TestUtils {
```

Comment:
please rename this class, has nothing to do with tests, only with json

Reply:
addressed.